content/docs/components/kustomize.md

```diff
@@ -0,0 +1,84 @@
++++
+title = "kustomize"
+description = "Information about kustomize as used in Kubeflow"
+weight = 20
++++
+
+Kubeflow makes use of [kustomize](https://kustomize.io/) to help customize YAML
+configurations.
+
+## Installing kustomize
+
+Make sure you have the minimum required version of kustomize:
+**{{% kustomize-min-version %}}** or later.
+
+The simplest way to install kustomize is to run `go get sigs.k8s.io/kustomize`.
+
+If you prefer installing it from source follow the steps below to install
+kustomize:
+
+1. Follow the [kustomize installation
+   guide](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md),
+   choosing the relevant options for your operating system. For example, if
+   you're on Linux:
+
+    * Set some variables for the operating system:
+
+        ```
+        export opsys=linux
+        ```
+
+    * Download the kustomize binary:
+
+        ```
+        curl -s https://api.github.com/repos/kubernetes-sigs/kustomize/releases/latest |\
+        grep browser_download |\
+        grep $opsys |\
+        cut -d '"' -f 4 |\
+        xargs curl -O -L
+        ```
+
+    * Move the binary:
+
+        ```
+        mkdir -p ${HOME}/bin
+        mv kustomize_*_${opsys}_amd64 ${HOME}/bin/kustomize
+        chmod u+x ${HOME}/bin/kustomize
+        ```
+
+1. Include the `kustomize` command in your path:
+
+      ```
+      export PATH=$PATH:${HOME}/bin/kustomize
+      ```
+
+## More about kustomize
+
+kustomize lets you customize raw, template-free YAML files for multiple
+purposes, leaving the original YAML untouched and usable as is.
+
+kustomize directories can be built using `kustomize build <kustomization_directory>`
+and applied using `kustomize build <kustomization_directory> | kubectl apply -f -`.
```

Comment:
- Do you want to mention the role of kustomization.yaml in more detail, or how Kubeflow has developed manifests to configure and deploy Kubeflow components?
- Is kustomize directly manipulated by the Kubeflow user, or are the commands embedded in scripts/templates?
- The directories contain manifest files (kustomization.yaml); the command takes the top level directory in the configuration file hierarchy. Does the user directly edit the manifest files to change the way Kubeflow is deployed?
- Do you want to provide a link to the Kubeflow manifest examples repo?

Reply:
+1, please include a link to https://github.com/kubeflow/manifests/blob/master/README.md

Reply:
I tried adding some of this detail but I don't have any hands on experience so it was difficult to add all the information. I'd appreciate any help!

Reply:
I think it is ok to leave out some of these details. The kfctl tool acts as a wrapper around the kustomize build commands, so we don't need to explain how kustomization.yaml is organized.

The user only needs to change manifest files if further customization is needed, in which case this page should help: https://www.kubeflow.org/docs/gke/customizing-gke/

Reply:
Thanks @richardsliu I added that link.